docs/core/tools/elevated-access.md

```diff
@@ -87,14 +87,14 @@ During development, you may need elevated access to test your application. This
 
 - Using generated executable (it provides the best startup performance):
 
-   ```bash
+   ```dotnetcli
```

Comment:
the ones that mix .NET CLI commands and others were purposely not called dotnetcli

Reply:
I think this is something you may need to discuss internally. I've done that based on https://github.com/dotnet/docs/issues/16462#issuecomment-570280349

Reply:
We discussed internally and thought we should continue to keep dotnetcli only for not mixed commands /cc @scottaddie